AAP-3481 Added new note and link to Chapt 8.4 in the Server Admin Guide (#1234)
RedHatInsights/red-hat-ansible-automation-platform-documentation: downstream/assemblies/central-auth/assembly-central-auth-add-user-storage.adoc

.Procedure
. Log in to {AAPCentralAuth} as an SSO admin user.
. From the navigation bar, select menu:Configure section[User Federation].

[NOTE]
====

> When using an LDAP User Federation in {RHSSOshort}, a group mapper must be added to the client configuration, ansible-automation-platform, to expose the identity provider (IDP) groups to the SAML authentication. Refer to link:https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/{RHSSOVers}/html-single/server_administration_guide/index#protocol-mappers[OIDC Token and SAML Assertion Mappings] for more information on SAML assertion mappers.

Author: Update to active voice here.

> When using an LDAP User Federation in {RHSSOshort}, a group mapper must be added to the client configuration, ansible-automation-platform, to expose the identity provider (IDP) groups to the SAML authentication. Refer to link:https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/{RHSSOVers}/html-single/server_administration_guide/index#protocol-mappers[OIDC Token and SAML Assertion Mappings] for more information on SAML assertion mappers.

Author: ```suggestion
When using an LDAP User Federation in {RHSSOshort}, you must add a group mapper to the client configuration, ansible-automation-platform, to expose the identity provider (IDP) groups to the SAML authentication. Refer to link:https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/{RHSSOVers}/html-single/server_administration_guide/index#protocol-mappers[OIDC Token and SAML Assertion Mappings] for more information on SAML assertion mappers.
```
